content/_components/sftp.md

```diff
@@ -5,7 +5,7 @@ section: Protocol components
 ---
 ---
 
-# SFTP Component [![CircleCI](https://circleci.com/gh/elasticio/sftp-component.svg?style=svg)](https://circleci.com/gh/elasticio/sftp-component)
+# SFTP Component 
```

Comment:
Please remove this line. The title is automatically generated from the header.

Reply:
@khanzadyan removed